engines/config-query-sparql/config/config-default.json

```diff
@@ -24,7 +24,7 @@
     "ccqs:config/merge-bindings-context/mediators.json",
     "ccqs:config/optimize-query-operation/actors.json",
     "ccqs:config/optimize-query-operation/mediators.json",
-    "ccqs:config/query-operation/actors.json",
+    "ccqs:config/query-operation/actors-4-2-0.json",
```

Comment:
This change will have to be made in config-default-v4-2-0.json (as a copy of config-default-v4-1-0.json).

And the configs in all query-sparql* packages will have to point to this new config-default-v4-2-0.json file.

All query-sparql* packages will also require swapping the old distinct actor for the new one in their package.json dependencies.